### PR TITLE
Add cluster delete instructions and reference to generic in gke docs

### LINKED ISSUE #637: Issue in page Deploying Scylla on GKE - missing parts
I would like to report an issue in page https://operator.docs.scylladb.com/v1.2/gke

### Problem
The example should also includes:
- How to open CQLSh (basic interaction)
- How to delete the cluster and clean resources once done

This is true for all examples (local, EKS)

### PR DESCRIPTION
**Pull request by rzetelskik**
**Description of your changes:**
- Add instructions on how to delete a GKE cluster in GKE docs.
- Add references to generic instructions on how to access the database in EWS and GKE docs.

**Which issue is resolved by this Pull Request:**
Resolves #637

### CODE REVIEW

File: docs/source/eks.md [hunk 1]
```diff
@@ -110,10 +110,14 @@ kubectl apply -f node-setup-daemonset.yaml
 
 Now you can follow the [generic guide](generic.md) to launch your Scylla cluster in a highly performant environment.
 
-#### Deleting a EKS cluster
+#### Accessing the database
 
-Once you are done with your experiments delete your cluster using following command:
+Instructions on how to access the database can also be found in the [generic guide](generic.md).
```
Comment (zimnx) [inline docs/source/eks.md:115]: any chance for direct link to subsection?

File: docs/source/gke.md [hunk 2]
```diff
@@ -153,4 +153,18 @@ sed -i "s/<gcp_region>/${GCP_REGION}/g;s/<gcp_zone>/${GCP_ZONE}/g" examples/gke/
 
 This will inject your region and zone into the cluster definition so that it matches the kubernetes cluster you just created.
 
+### Installing the Scylla Operator and Scylla
+
 Now you can follow the [generic guide](generic.md) to install the operator and launch your Scylla cluster in a highly performant environment.
+
+#### Accessing the database
+
+Instructions on how to access the database can also be found in the [generic guide](generic.md).
```
Comment (zimnx) [inline docs/source/gke.md:162]: ditto

File: docs/source/eks.md (hunk 1 above)
Comment (rzetelskik) [reply docs/source/eks.md:115]: Tried the direct link and it didn't work locally so I kept it that way

**Review by zimnx — APPROVED** [review]
LGTM, thanks

**Review by zimnx — APPROVED** [review]

**Pull request merged**